File: app/src/main/java/info/plateaukao/einkbro/browser/AlbumController.java
package info.plateaukao.einkbro.browser;

import info.plateaukao.einkbro.view.Album;

@SuppressWarnings("unused")
public interface AlbumController {

    Album getAlbum();

    String getAlbumTitle();

    void setAlbumTitle(String title);

    void activate();

    void deactivate();

    String getAlbumUrl();

    void pauseWebView();

    void resumeWebView();
}

File: app/src/main/java/info/plateaukao/einkbro/browser/AlbumController.kt
package info.plateaukao.einkbro.browser

import info.plateaukao.einkbro.view.Album

interface AlbumController {
    val album: Album
    var albumTitle: String
    val albumUrl: String
    var initAlbumUrl: String

    var isTranslatePage: Boolean

    var isAIPage: Boolean

    fun activate()
    fun deactivate()

    fun pauseWebView()
    fun resumeWebView()
}